interfaces/apparmor/backend.go

```diff
@@ -250,6 +280,7 @@ func snapConfineProfileName(snapName string, rev snap.Revision) string {
 //
 // Additionally it will cleanup stale apparmor profiles it created.
 func (b *Backend) setupSnapConfineReexec(info *snap.Info) error {
+	logger.Noticef("setup snap confine reexec")
```

Comment:
do we get a matching reload profiles log from this helper? otherwise is a bit confusing that the two paths produce different log traces

Reply:
dropped